apps/application-system/form/src/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Umsóknir - Ísland.is</title>
+
+    <meta name="title" content="Umsóknir - Ísland.is" />
+
+    <meta
+      name="description"
+      content="Umsóknakerfi bætir opinbera þjónustu með því að auðvelda notendum sjálfsafgreiðslu. Stofnanir geta þá tekið á móti umsóknum og eyðublöðum með stafrænum hætti og gefið um leið skýrari upplýsingar um framgang og niðurstöðu."
+    />
+    <meta property="og:title" content="Umsóknir - Ísland.is" />
+    <meta
+      property="og:description"
+      content="Umsóknakerfi bætir opinbera þjónustu með því að auðvelda notendum sjálfsafgreiðslu. Stofnanir geta þá tekið á móti umsóknum og eyðublöðum með stafrænum hætti og gefið um leið skýrari upplýsingar um framgang og niðurstöðu."
+    />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="http://island.is/umsoknir/" />
+    <meta
+      property="og:image"
+      content="https://island.is/island-fb-1200x630.png"
+    />
+    <meta property="og:image:width" content="1200" />
+    <meta property="og:image:height" content="630" />
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="http://island.is/umsoknir/" />
+    <meta property="twitter:title" content="Umsóknir - Ísland.is" />
+    <meta
+      property="twitter:description"
+      content="Umsóknakerfi bætir opinbera þjónustu með því að auðvelda notendum sjálfsafgreiðslu. Stofnanir geta þá tekið á móti umsóknum og eyðublöðum með stafrænum hætti og gefið um leið skýrari upplýsingar um framgang og niðurstöðu."
+    />
+    <meta
+      property="twitter:image"
+      content="https://island.is/island-fb-1200x630.png"
+    />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+
+    <script
+      async
+      defer
+      data-domain="island.is/umsoknir"
+      src="https://plausible.io/js/plausible.js"
+    ></script>
+
+    <script id="__SI_ENVIRONMENT__" type="application/json">
+      {}
+    </script>
+  </head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>
```

Comment:
What is this file and why does it pop up again and again?

Reply:
no idea why this is happenning

Reply:
I think we should remove it, just in case :D